### DISCUSSION TITLE
Welcome to XFlow Discussions!

### DISCUSSION THREAD

**Discussion by lyn-boyu** [antvis/XFlow#51] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZerooreZ123** [comment]
在DAG 解决方案中，如何改变连线的样式（比如颜色）
